re-frisk/src/re_frisk/ui.cljs

```diff
@@ -56,19 +56,30 @@
     (fn []
       (when-not @ext-opened?
         (let [left (or (utils/normalize-draggable (:x @drag/draggable))
-                       (- js/window.innerWidth 30))]
+                       (- js/window.innerWidth 30))
+              handle-toggle (fn [] 
+                              (when-not (utils/closed? left)
+                                     (swap! db/tool-state assoc :latest-left (- js/window.innerWidth left)))
+                              (swap! drag/draggable assoc :x (- js/window.innerWidth
+                                                                (if (utils/closed? left) @latest-left 30))))
```

Comment:
im not sure if latest-left should be a reaction, so we could move this function outside , and use tool-state directly in that case we could addListener only once outside as well

Reply:
yeah if we we want to hide UI button, we need to addListener outside anyway

Reply:
im working on the new cool feature in re-frisk now, so I can finish your PR

Reply:
Very nice, would be a great feature to have. 👍